src/reducers/catalogReducer.js

```diff
@@ -40,12 +40,9 @@ const catalogReducer = produce((draft, action) => {
       return;
 
     case types.CLUSTER_LOAD_APP_README_SUCCESS:
-      if (!draft.items[action.catalogName]) return;
```

Comment:
I'll put this back.

It's a bit clearer what the intent is in such a long line. And also easier syntax to get right.